src/main/docker/Dockerfile

```diff
@@ -38,7 +40,7 @@ WORKDIR /opt/artemis
 ARG WAR_PATH=build/libs
 COPY $WAR_PATH/*.war Artemis.war
 
-COPY bootstrap.sh /bootstrap.sh
+COPY ${BOOTSTRAP_SCRIPT} /bootstrap.sh
```

Comment:
Out of my gut feeling I would rather mount and overwrite the bootstrap.sh inside the container with your version of it than inserting it via the ARG.

Reply:
Also changed into separate image